eo-runtime/src/main/java/org/eolang/PhPackage.java

```diff
@@ -35,6 +35,7 @@
  * @since 0.22
  */
 @Versionized
+@SuppressWarnings({"PMD.TooManyMethods", "PMD.ConstructorShouldDoInitialization"})
```

Comment:
@c71n93 I believe, you can easily fix this error (that PMD complains about), instead of suppressing it